openedx/core/djangoapps/user_api/management/tests/test_email_opt_in_list.py

```diff
@@ -393,7 +412,11 @@ def _assert_output(self, output, *args, **kwargs):
         for user, course_id, opt_in_pref in args:
             self.assertIn({
                 "email": user.email.encode('utf-8'),
-                "full_name": user.profile.name.encode('utf-8'),
+                "full_name": (
+                    user.profile.name.encode('utf-8')
+                    if hasattr(user, 'profile')
```

Comment:
@adampalay If I understand this correctly, do you want to update the following to use `kwargs`?
```
user.profile.name.encode('utf-8')
if hasattr(user, 'profile')
else ''
```

We are passing these three params as args to the method, For example.
```
self._assert_output(
 output,
 (user, courses_id, False),
 (user, courses_id, False),
)
``` 
We can not use `kwargs` to condition user.